spec/mailers/previews/devise/mailer_preview.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# Preview all emails at http://localhost:3000/rails/mailers/devise_mailer
+module Devise
+  # :nodoc:
+  class MailerPreview < ActionMailer::Preview
+    def confirmation_instructions
+      Devise::Mailer.confirmation_instructions(User.last, "faketoken")
+    end
+
+    def email_changed
+      Devise::Mailer.email_changed(User.last, User.last.email)
```

Comment:
this is not working: http://localhost:3000/rails/mailers/devise/mailer/email_changed